src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/CaseActionsForCaseWorkerController.java

```diff
@@ -301,7 +329,7 @@ public ResponseEntity<CCDCallbackResponse> amendCaseDetails(
             var defaultValues = getPostDefaultValues(caseDetails);
             log.info("Post Default values loaded: " + defaultValues);
             defaultValuesReaderService.getCaseData(caseData, defaultValues);
-            caseManagementForCaseWorkerService.dateToCurrentPosition(caseData);
+            caseManagementForCaseWorkerService.setDateToCurrentPosition(caseData);
             caseManagementForCaseWorkerService.setNextListedDate(caseData);
```

Comment:
Move this method to the service class perhaps?

Reply:
@GauravChawlaMOJ this method is literally reusing what was in a controlled method. I think it is better placed here as moving it would introduce at least dependency on three other services into the CaseManagementForCaseWorkerService.